File: c++-problem-solving/swexpertacademy/odd-numbers-plus-2072.cpp
#include <iostream>
#include "solutions.h"

using namespace std;

void odd_numbers_plus_2072() {

    int time;
    cin >> time;

    for(int i=0; i<time; i++) {
        int sum = 0;

        for(int j=0; j<10; j++) {
            int input;
            cin >> input;

            if(input % 2 == 1) {
                sum += input;
            }
        }
        cout << "#" << i + 1 << " " << sum << endl;
    }
}

File: JavaProblemSolving/src/swexpertacademy/OddNumbersPlus2072.java
package swexpertacademy;

import java.util.Scanner;

public class OddNumbersPlus2072 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int t = scanner.nextInt();

        for(int i=1; i<=t; i++){
            int sum = 0;

            for(int j=0; j<10; j++){
                int input = scanner.nextInt();
                if(input % 2 == 1){
                    sum += input;
                }
            }

            System.out.println("#"+i+" "+sum);
        }
    }
}
